samza-kafka/src/test/java/org/apache/samza/system/kafka/TestKafkaSystemFactoryJava.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.samza.config.ApplicationConfig;
```

Comment:
I don't see any test that satisfy `kafka properties doesn't support replication.factor when creating topic` as per Jira description.